data/presets/presets/barrier/kerb.json

```diff
@@ -0,0 +1,14 @@
+{
+    "icon": "wheelchair",
+    "fields": [
+        "access"
+    ],
+    "geometry": [
+        "point",
```

Comment:
For this preset, change "point" to "vertex".. (Points are standalone nodes, and vertices are nodes that exist along a parent way)

Reply:
sounds good i'll make these tweaks 👍